packages/datatrak-web/src/routes/Routes.tsx

```diff
@@ -58,6 +59,7 @@ const AuthViewLoggedInRedirect = ({ children }) => {
 export const Routes = () => {
   return (
     <RouterRoutes>
+      <Route path="/offline" element={<OfflinePage />} />
```

Comment:
I'm not 100% sure how the offline page will be used yet so I've just put it here